docker-compose.local.yml

```diff
@@ -0,0 +1,20 @@
+version: "2.8"
+services:
+  db:
+    container_name: dfe-complete-sql-server
+    image: mcr.microsoft.com/mssql/server:2019-latest
+    env_file: .env.database
+    ports:
+      - 1433:1433
+    volumes:
+      - sql-server-data:/var/opt/mssql
+  redis:
+    image: redis:6
+    ports:
+      - 6379:6379
+    volumes:
+      - redis-data:/data
+
+volumes:
+  sql-server-data:
+  redis-data:
```

Comment:
Is it intentional that we have a new `docker-compose.local` which does this? It sort of makes sense for local dev to have this but not prod (because we don't yet know how that will have Redis provisioned), but this also means we can't do things like run tests against it unless the test runner provisions its own.

Reply:
It is intentional - my understanding is we have devs who use the script, those that run everything in Docker and devs that run hybrid.

The scripts install Redis natively so we only want to start the MSSQL server in Docker as it cannot be run natively on MacOS so we have `docker-compose.database.local.yml`

For all in Docker folks we have `docker-compose.yml` which starts everything

And for folks that want to run hybrid (i.e. me! :)) have `docker-compose.local.yml` to start the backing services but not the application.

In CI we use the `docker-compose.ci.yml` which will provision the Redis instance.

Does that anwswer the question?